project/AppDependencies.scala

```diff
@@ -1,45 +1,38 @@
+import play.sbt.PlayImport.ws
 import sbt._
-import play.core.PlayVersion
-import play.sbt.PlayImport._
 
 object AppDependencies {
-  
-  lazy val scope: String = "test"
 
-  val silencerVersion = "1.7.1"
-  val bootstrapVersion = "5.24.0"
+  val silencerVersion = "1.7.9"
+  val bootstrapVersion = "6.3.0"
   val akkaVersion = "2.6.19"
 
   val compile: Seq[ModuleID] = Seq(
     ws,
-    "uk.gov.hmrc" %% "bootstrap-backend-play-28" % bootstrapVersion,
-    "uk.gov.hmrc" %% "domain" % "8.1.0-play-28",
-    "uk.gov.hmrc" %% "http-caching-client" % "9.6.0-play-28",
-    "uk.gov.hmrc" %% "tabular-data-validator" % "1.4.0",
-    "org.scala-lang.modules" %% "scala-xml" % "1.3.0",
-    "org.scala-lang.modules" %% "scala-parser-combinators" % "2.1.0",
```

Comment:
is "scala-parser-combinators" not needed anymore?

Reply:
Yep it's not used anywhere